tests/models/test_model.py

```diff
@@ -149,9 +189,14 @@ def test_model_info():
         x = _dataframe_from_json(path)
         assert x.to_dict(orient="records")[0] == input_example
 
-        assert model_info.signature_dict == sig.to_dict()
+        model_signature = model_info_fetched.signature
+        assert model_info.signature_dict == sig.to_dict() == model_signature.to_dict()
 
-        assert Version(model_info.mlflow_version) == Version(loaded_model.mlflow_version)
+        assert (
+            Version(model_info.mlflow_version)
+            == Version(loaded_model.mlflow_version)
+            == Version(model_info_fetched.mlflow_version)
+        )
```

Comment:
```suggestion
        assert (
           model_info.mlflow_version
            == loaded_model.mlflow_version
            == model_info_fetched.mlflow_version
        )
```

I'm not sure why the original code uses `Version`, but I think we can remove it.